src/js/rx/components/Breadcrumbs.jsx

```diff
@@ -0,0 +1,38 @@
+import _ from 'lodash';
+import { Link } from 'react-router';
+import React from 'react';
+
+class Breadcrumbs extends React.Component {
+  render() {
+    const { location: { pathname }, prescription } = this.props;
+
+    const crumbs = [
+      <Link href="/" key="home">Home</Link>,
```

Comment:
I guess the only links that should be `Link` are the 'Prescription refills' crumbs that go to the app's root.